Protos/V1/events_common.proto

```diff
@@ -4,14 +4,31 @@ package armonik.api.grpc.v1.events;
 
 import "result_status.proto";
 import "task_status.proto";
+import "tasks_filters.proto";
+import "results_filters.proto";
 
 option csharp_namespace = "ArmoniK.Api.gRPC.V1.Events";
 
+
+/**
+ * Represents the events that can be returned in the EventSubscriptionResponse
+ */
+enum EventsEnum {
+  EVENTS_ENUM_NEW_TASK = 0; /** New task */
+  EVENTS_ENUM_TASK_STATUS_UPDATE = 1; /** Task status update */
+  EVENTS_ENUM_NEW_RESULT = 2; /** New restult */
+  EVENTS_ENUM_RESULT_STATUS_UPDATE = 3; /** Result status update */
+  EVENTS_ENUM_RESULT_OWNER_UPDATE = 4; /** Result owner update */
+}
+
 /**
  * Request to subscribe to the event stream.
  */
 message EventSubscriptionRequest {
   string session_id = 1; /** Id of the session that will be used to subscribe events for. **/
```

Comment:
@lemaitre-aneo do we keep the session Id and put it in the filter internally or do we remove it ?

Reply:
I'm not sure. It could be embedded into the filters, but this would allow to listen on everything! I think we should keep this tied to a session.